gateway/rpc_storage_handler.go

```diff
@@ -1021,6 +1023,8 @@ func (r *RPCStorageHandler) ProcessKeySpaceChanges(keys []string, orgId string)
 			case OauthClientAdded, OauthClientUpdated, OauthClientRemoved:
 				OauthClients[splitKeys[0]] = action
 				notRegularKeys[key] = true
+			case deleteAPICache:
```

Comment:
If you add a stringer for NotificationCommand, you could just use NoticeAPIDeleteCache.String()